app/src/main/kotlin/com/wire/android/WireApplication.kt

```diff
@@ -254,7 +254,16 @@ class WireApplication : BaseApp() {
                 .isAppVisibleFlow()
                 .filter { isVisible -> isVisible }
                 .collect {
-                    AnonymousAnalyticsManagerImpl.sendEvent(AnalyticsEvent.AppOpen)
+                    val currentSessionResult = coreLogic.get().getGlobalScope().session.currentSessionFlow().first()
+                    val isTeamMember = if (currentSessionResult is CurrentSessionResult.Success) {
+                        coreLogic.get().getSessionScope(currentSessionResult.accountInfo.userId).team.isSelfATeamMember()
+                    } else {
+                        null
```

Comment:
are you sure about `null`, maybe `false`? If user is not logged in - user is not in a team :)

Reply:
If we pass `false`, it means we have a personal user logged in. So better to keep undefined and pass null to not send `isTeamMember` segmentation at all if there is no user logged in.